Italian layouts

nicoursi opened the pull request:
Fixes #1736

Created an Italian with symbols layout.
I also added popup domains to the it locale

menny commented on addons/languages/italian/pack/src/main/res/xml/italian_autotext.xml:
```diff
@@ -20,6 +20,25 @@
+
+    <!-- autotext needs to be improved. e.g if the user types (after a dot) Cone, it will be replaced by come and not Come  -->
+
+    <word src="cone">come</word>
```
this is great. thanks

menny commented on addons/languages/italian/pack/src/main/res/xml/italian_qwerty_with_symbols.xml:
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
```
I want to give @martholomew the option to review this.

nicoursi commented:
Can we get rid of Italian_querty?
If so, what name should we keep? Italian_querty or italian_querty_with_symbols?

menny commented:
can you add detail about this change to https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/italian/apk/src/main/play/release-notes/en-US/alpha.txt

nicoursi commented:
> can you add detail about this change to https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/italian/apk/src/main/play/release-notes/en-US/alpha.txt

done

nicoursi commented:
I added another important commit. In italian we don't have the word `po`.  We hve the word `poco` that gets truncated and becomes `po'`.
In the dictionary `aosp.combined.gz` there is the following entry:

` word=po,f=136,flags=,originalFreq=136`

Whenever the user wants to write `po'`, he'll only get suggestions for `po`, even if `po` doesn't really exist, and it's considered a mistake in italian.

Commit #2420b96 fixes this by replacing the entry above with the following one:

`word=po',f=136,flags=,originalFreq=136`

It's tested and working. I also added this change in release notes [release notes](https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/addons/languages/italian/apk/src/main/play/release-notes/en-US/alpha.txt)

nicoursi commented:
@menny 

I added another important commit. In italian we don't have the word po. We hve the word poco that gets truncated and becomes po'.
In the dictionary aosp.combined.gz there is the following entry:

word=po,f=136,flags=,originalFreq=136

Whenever the user wants to write po', he'll only get suggestions for po, even if po doesn't really exist, and it's considered a mistake in italian.

Commit #2420b96 fixes this by replacing the entry above with the following one:

word=po',f=136,flags=,originalFreq=136

It's tested and working. I also added this change in release notes release notes

menny commented:
Thanks! I'm having difficulties finding time to review PRs at the moment. This looks like an important PR. Sorry for the delay
